Presentation.Web/app/components/it-advice/it-advice-modal-view.html

```diff
@@ -23,8 +23,8 @@ <h3 class="modal-title pull-left">{{advisName}}</h3>
                 </div>
 
                 <div class="form-group">
-                    <label for="contract-note" class="control-label  small">Til modtager via email</label>
-                    <input id="toExternal" type="email" name="toExternal" data-ng-disabled="!isEditable()" placeholder="Skriv email" class="form-control input-sm" data-ng-model="externalTo" data-element-type="adviceEmailToInput">
+                    <label for="contract-note" class="control-label small">Til modtager via email</label>
```

Comment:
mener for=".." feltet er rettet i master, men ellers er "contract-note" forkert (skal være "toExternal")